docs/platform.rst

```diff
@@ -16,6 +16,10 @@ InfluxDB®,
 Grafana®,
 and M3 - all available in more than 80 regions around the world on AWS, GCP, Microsoft Azure, DigitalOcean, and UpCloud cloud platforms.
 
+What is the Aiven platform?
```

Comment:
I think the title "What is the Aiven platform?" doesn't reflect  the content of the section.
Maybe we can use something like "Latest improvements" or "What's new" or something else reflecting the info on changelog. What do you think?

Reply:
I think I copied and did not change, my bad. Thanks for correcting it!